lib/matplotlib/axes/_axes.py

```diff
@@ -4188,7 +4188,8 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
             sequence of floats, as required by
             :class:`~matplotlib.collections.RegularPolyCollection`.
 
-        edgecolors : {'none'}, mpl color, color sequence, optional, default is 'none'
+        edgecolors : {'none'}, mpl color, color sequence, optional,
+        default is 'none'
```

Comment:
Can you indent this line and drop the coma at the end of the previous line?  My guess is that sphinx is going to complain about this.